### DISCUSSION TITLE
Is the style function with multiple args more performant than an array of tw template tags?

### DISCUSSION THREAD

**Discussion by erquhart** [jaredh159/tailwind-react-native-classnames#311] [Q&A] [answered]
I prefer the template tag API, but have been changing them to `tw.style()` when conditions come into play. It would be nice to just use one approach everywhere, but I don't want to use `tw.style()` everywhere. One solution is to pass an array of template tags, as react native arrays support conditionals really well.

I haven't looked too deep into twrnc performance - is there a penalty to one approach over the other?

**Comment by jaredh159** [comment, marked as answer]
the template tag de-sugars to some calls to `tw.style()` so technically, `tw.style()` would be a smidge faster, but it shouldn't matter in practice. twrnc is heavily cached and performs really well, it's unlikely that would be a source of performance problems.

**Reply by erquhart** [reply]
Alright, that's good to know. Thank you, and thanks for this amazing lib! Been using for over a year, love it.
